### ISSUE TITLE
Imports Duplicate Locations

### ISSUE THREAD

**Issue by betovicentini** [opendatabio/opendatabio-r#14]
This is separating this issue from #11.
- The import location API is still importing duplicated Locations (countries so far tested).
- About the Unique KEY reference in the location table raised in #11 , I do have it correctly in my localhost installation:
```
  UNIQUE KEY `locations_name_parent_id_unique` (`name`,`parent_id`);
```
- And he duplicate importation also happens in the USP installation, so this seem not to be the problem.
- I tested this with the following command in R:
```
#TESTE BOTH ADDRESSES WITH MY TOKEN
base_url="http://localhost/opendatabio/api"
base_url2="http://opendatabio.ib.usp.br/opendatabio/api"
library(raster)
library(opendatabio)
lv <- 0
ocrt <- try(getData('GADM', country="ECU", level=lv),silent = T)
odb_import_locations(sp_to_df(ocrt), odb_config(base_url=base_url, token = token))
```
- This file size is ```Sending ODB request (filesize =  4617720 )```

**Comment by andrechalom** [comment]
Right!! This is truly a bug affecting country data, because of the way MySQL handles composite keys with null values... You can enter multiple countries with the same name, because the parent for all of them is `NULL`. 

I'll try to add a "hidden" World entry with Id = 0, so all countries will be children of the same parent. This will fix the issue.

**Comment by andrechalom** [comment]
(This commit also fixes a bug in editing Country level locations)

**Comment by andrechalom** [comment]
@birutaibm please check the usage example here ^^^

**Comment by andrechalom** [comment]
Fixed in opendatabio #190
